Rewrite the intro to IOTA. (#565)
iota-community/iota-wiki: internal/learn/about-iota/coordinator.md

---
id: coordinator

> title: Coordinator

Reviewer: ```suggestion
title: The Coordinator
```

> description: What is Coordinator in the IOTA network.

Reviewer: ```suggestion
description: The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages.
```
and please add an image as well

---

> # Coordinator

Reviewer: ```suggestion
# The Coordinator
```

:::note

> Coordinator is a temporary solution. IOTA will remove Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).

Reviewer: ```suggestion
The Coordinator is a temporary solution. IOTA will remove the Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
```

:::

> Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.

Reviewer: ```suggestion
The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are only considered for confirmation when they are directly or indirectly referenced by a milestone that nodes have validated.
```

Reviewer: this could use a link to What is Chrysalis , https://wiki.iota.org/chrysalis-docs/introduction/what_is_chrysalis#new-milestone-selection-algorithm

Author: I don't like it. Crysalis is the reality for the long time now and the article still talks about it as of a new feature. I believe that not a single newcomer cares about the fact the current revision of IOTA is called Crysalis or how awesome it used to be compared to IOTA 1.0.

> To allow the nodes to recognize valid milestones, all nodes that participate in the same IOTA network are configured with the Ed25519 signatures of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.

Reviewer: ```suggestion
To allow the nodes to recognize valid milestones, all nodes participating in the same IOTA network are configured with the [Ed25519 signatures](https://wiki.iota.org/chrysalis-docs/introduction/what_is_chrysalis#ed25519-signature-scheme) of a coordinator they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.
```

Author: I've removed the mention of Ed25519. Same concern as with crysalis in general.

> To ensure that new messages always have a chance of being confirmed, Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This way ensures that nodes can compare the indexes of their milestones to check if they are synchronized with the rest of the network.

Reviewer: ```suggestion
To ensure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones signed with those signatures regularly (every 10 seconds). This ensures that nodes can compare the indexes of their milestones to check if they are synchronized with the rest of the network.
```

---
id: coordinator
title: The Coordinator
description: What is the Coordinator in the IOTA network.
image: /img/iota-wiki.png
keywords:
- explanation
---

# The Coordinator

:::note

The Coordinator is a temporary solution. IOTA will remove the Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).

:::

> Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.

Reviewer 2: ```suggestion
The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
```

> To allow the nodes to recognize valid milestones, all IOTA nodes on the same network are configured with the signatures of a coordinator node which they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted coordinator did sign them.

Reviewer 2: ```suggestion
To allow the nodes to recognize valid milestones, all IOTA nodes on the same network are configured with the signatures of a coordinator node which they trust. By knowing those signatures, nodes can validate the signatures in issued milestones to verify whether the trusted Coordinator did sign them.
```
